UPGRADE-1.0.md

```diff
@@ -4,8 +4,10 @@ UPGRADE FROM 0.13 to 1.0
 # Table of Contents
 
 - [Customize the cursor encoder of the edges of a connection](#customize-the-cursor-encoder-of-the-edges-of-a-connection)
-- [Change arguments of `TypeGenerator`](#change-arguments-of-typegenerator)
+- [Change arguments of `TypeGenerator`](#change-arguments-of-typegenerator-class)
 - [Add magic `__get` method to `ArgumentInterface` implementors](#add-magic-__get-method-to-argumentinterface-implementors)
+- [Rename `GlobalVariables` to `GraphQLServices`](#)
+- [Change `overblog_graphql.global_variable` tag]()
 
```

Comment:
Missing parts for resolver, mutation, all public tags, interfaces, classes ... changes (just a reminder)

Reply:
@mcg-web same here, I'll write the UPDATE documentation.